pyproject.toml

```diff
@@ -77,3 +68,29 @@ profile = "black"
 [tool.pytest.ini_options]
 addopts = "--strict-markers --strict-config"
 xfail_strict = true
+
+[tool.tox]
+legacy_tox_ini = """
+[tox]
+envlist = py{36,37,38}
+isolated_build = True
+
+[testenv:py{36,37,38}]
+# from https://github.com/python-poetry/poetry/issues/1941
+# it appears there is no easy way to access [tool.poetry.dev-dependencies]
```

Comment:
I dont know much about tox or tox+poetry config, but maybe see https://python-poetry.org/docs/faq/#is-tox-supported

I would imagine we have to tell tox to run `poetry install` somehow?

Reply:
Also should add mypy here somehow perhaps?

Reply:
yep mentioned this above.
In general though this is all working.
No you don't need to call `poetry install`, the installs are part of the build process